app/helpers/api/plans_helper.rb

```diff
@@ -19,7 +19,7 @@ def default_application_plan_data(service, plans)
   end
 
   def application_plans_data(plans)
-    plans.alphabetically
+    plans.order(name: :asc)
```

Comment:
Why does not that works? https://github.com/3scale/porta/blob/b2fd9c02596ce109f17d692475384c4aae7259d2/app/models/plan.rb#L130

Reply:
So the problem was I was passing 2 different collections through the same method. Once is an array and the other a ActiveRecord collection.
Moreover, `alphabetically` is redundant because plans are being ordered by the controller (since it depends on the query params)